models/stp_if_pol.go

```diff
@@ -0,0 +1,100 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ciscoecosystem/aci-go-client/container"
+)
+
+const StpIfPolClassName = "stpIfPol"
+
+type SpanningTreeInterfacePolicy struct {
+	BaseAttributes
+	SpanningTreeInterfacePolicyAttributes
+}
+
+type SpanningTreeInterfacePolicyAttributes struct {
+	Name string `json:",omitempty"`
+
+	Annotation string `json:",omitempty"`
+
+	Ctrl string `json:",omitempty"`
+
+	NameAlias string `json:",omitempty"`
+}
+
+func NewSpanningTreeInterfacePolicy(stpIfPolRn, parentDn, description string, stpIfPolattr SpanningTreeInterfacePolicyAttributes) *SpanningTreeInterfacePolicy {
+	dn := fmt.Sprintf("%s/%s", parentDn, stpIfPolRn)
+	return &SpanningTreeInterfacePolicy{
+		BaseAttributes: BaseAttributes{
+			DistinguishedName: dn,
+			Description:       description,
+			Status:            "created, modified",
+			ClassName:         StpIfPolClassName,
+			Rn:                stpIfPolRn,
+		},
+
+		SpanningTreeInterfacePolicyAttributes: stpIfPolattr,
+	}
+}
+
+func (stpIfPol *SpanningTreeInterfacePolicy) ToMap() (map[string]string, error) {
+	stpIfPolMap, err := stpIfPol.BaseAttributes.ToMap()
+	if err != nil {
+		return nil, err
+	}
+
+	A(stpIfPolMap, "name", stpIfPol.Name)
+
+	A(stpIfPolMap, "annotation", stpIfPol.Annotation)
```

Comment:
Annotation is a generic attribute. Can we move it to somewhere it will be inherited by everyone?

Reply:
Done as per discussion